client/src/productList/RealEstatePreview.js

```diff
@@ -0,0 +1,11 @@
+export default function RealEstatePreview(props) {
+  return (
+    <div>
+      <p>Nom: {props.realEstate.name}</p>
+      <p>Description: {props.realEstate.description}</p>
+      <p>Prix: {props.realEstate.price}</p>
+      <p>Adresse: {props.realEstate.address}</p>
+      <img src={props.realEstate.images[0]}/>
```

Comment:
Tu peux déstructurer `realEstate` pour réduire la complexité :
```suggestion
export default function RealEstatePreview({ realEstate }) {
  return (
    <div>
      <p>Nom: {realEstate.name}</p>
      <p>Description: {realEstate.description}</p>
      <p>Prix: {realEstate.price}</p>
      <p>Adresse: {realEstate.address}</p>
      <img src={realEstate.images[0]}/>
```